napari/layers/labels/labels.py

```diff
@@ -549,35 +549,12 @@ def show_selected_label(self, filter):
     @property
     def brush_shape(self):
         """str: Paintbrush shape"""
```

Comment:
```suggestion
```
I think we can get rid of the whole public `layer.brush_shape` attribute and `layer._brush_shape` too